tests/Tag/ATest.php

```diff
@@ -78,15 +97,16 @@ public function dataTarget(): array
      */
     public function testTarget(string $expected, ?string $contextName): void
     {
-        self::assertSame($expected, (string)A::tag()->target($contextName));
+        $this->assertSame($expected, (string)A::tag()->target($contextName));
     }
 
     public function testImmutability(): void
     {
         $tag = A::tag();
-        self::assertNotSame($tag, $tag->href(null));
-        self::assertNotSame($tag, $tag->url(null));
-        self::assertNotSame($tag, $tag->mailto(null));
-        self::assertNotSame($tag, $tag->target(null));
+        $this->assertNotSame($tag, $tag->content(''));
+        $this->assertNotSame($tag, $tag->href(null));
+        $this->assertNotSame($tag, $tag->url(null));
+        $this->assertNotSame($tag, $tag->mailto(null));
+        $this->assertNotSame($tag, $tag->target(null));
```

Comment:
Need test of method `rel()` on immutability.

Reply:
Done.